### DISCUSSION TITLE
[chart]option to not plot added metrics in chart, only display values in tooltip on hover

### DISCUSSION THREAD

**Discussion by daniel10012** [apache/superset#19177] [Ideas]
For example in Barcharts, I am looking to be able to see a metric by hovering on the barchart, but without it being graphed

![image](https://user-images.githubusercontent.com/10360991/109882818-735e9500-7c48-11eb-81d5-59f064582b55.png)

Here I would like the Count not to appear in the bars but only in tooltip

**Comment by daniel10012** [comment]
@junlincc

**Comment by junlincc** [comment]
@daniel10012 try going to Customize tab and uncheck the Bar value box? 

<img width="804" alt="Screen Shot 2021-03-08 at 9 04 15 PM" src="https://user-images.githubusercontent.com/67837651/110421189-db9ae580-8051-11eb-8cd2-af6c6f0707c3.png">

**Comment by daniel10012** [comment]
@junlincc this just adds/removes the bar value at the top but not the fact that it is graphed ?

**Comment by junlincc** [comment]
hm I don't understand what you mean by ‘in the graph’😳 we do need to fix all the overlapping on the top! @daniel10012

**Comment by villebro** [comment]
@junlincc I think the idea is to add metrics that only show up on the tooltip, not on the chart (this is available e.g. in Tableau). This is fairly simple to implement (good starter task).

**Comment by junlincc** [comment]
ooook i got it now. It is very common in BI tools. 
<img width="552" alt="Screen Shot 2021-03-09 at 10 48 18 AM" src="https://user-images.githubusercontent.com/67837651/110522442-c19aea80-80c5-11eb-8b4d-b786c1a59dcb.png">

How should we best approach it from UI?

In Tableau, user simply drag measures into the tooltip field, which is separate from rows(x) and columns(y)
In Superset, if we start introducing different dropping fields(tooltip, size and label etc), our current layout will be changed a lot. But i dont think we should only limit the customization to tooltip. 
<img width="173" alt="Screen Shot 2021-03-09 at 10 52 28 AM" src="https://user-images.githubusercontent.com/67837651/110522254-8ef0f200-80c5-11eb-9ce0-47d90d4070cf.png">

cc @kgabryje @zhaoyongjie let's brainstorm 

@mihir174 @steejay would love to get your thoughts on it, happy to provide more details to you.

**Comment by ValentinC-BR** [comment]
This would be a great enhancement !

This could be done in the UI via an additional section "tooltip" with a "metrics" input for instance.

**Comment by junlincc** [comment]
@ValentinC-BR right, that will would in Customize tab, assuming we are keeping the current tab layout.

**Comment by mihir174** [comment]
We could add an additional section for tooltip metadata like this - 

<img width="288" alt="Line Chart - Data (Default)" src="https://user-images.githubusercontent.com/64227069/111694049-9a8f9780-87ee-11eb-94e0-7a2c45cf469f.png">

But I think we should think about chart customization more holistically, and identify other additional input fields we might want to have eg. label, color etc like Tableau

**Comment by olof-dev** [comment]
Was there any progress on this? It'd be a fantastic feature, particularly for line charts. Each line in the chart would show you some particular main metric of interest, and then other information you might be curious about would be available in the tooltip.

**Comment by szhem** [comment]
Very nice feature to have. 
Currently I have to duplicate all the auxiliary metrics within an additional pivot-table chart eating valuable dashboard space.
Having an ability to show additional metrics within a tooltip would save dashboard space and would decrease a number of queries to the DWH.

**Comment by dereksmart** [comment]
+1, just spent an hour trying to do this 😄 would be very nice to have.

**Reply by devim1982** [reply]
+1 for this feature, would be ideal to be able to populate context where tooltip can be filled in via dataset column

**Comment by davidriverapericas** [comment]
+1, it would be great to have this feature

**Comment by mazur162** [comment]
+1 for this feature

**Reply by mazur162** [reply]
Are there any updates?
